card_house/controller.py

```diff
@@ -0,0 +1,73 @@
+import os
+from typing import List, Dict, Any
+
+from card_house.cards_exporter.cards_exporter.icards_exporter import ICardsExporter
+from card_house.db.error import CardExists
+from card_house.infrastructure.cards_creators.db_cards_creator import DBCardsCreator
+from card_house.infrastructure.cards_creators.json_to_cards_creator import JsonToCardsCreator
+from card_house.infrastructure.entities.db_card import DBCard, CardKeys
+from card_house.db.iclient_db import IClientDB
+
+from flask import Flask, request
+
+
+CARDS_SERIALIZED = Dict[str, List[Dict[str, str]]]
```

Comment:
Is this according to convention? It's not a const it's a type. 
If it is, make sure to look for all the places we used type aliasing and change them to this convention.

Reply:
its unused anyway, removed